hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -523,6 +531,32 @@ public static OmKeyInfo prepareKeyForRecover(OmKeyInfo keyInfo,
     }
   }
 
+  /**
+   * Get the valid base object id given the transaction id.
+   * @param id of the transaction. This value cannot exceed 2^40 - 1 as
+   *           out of the 64 bits for a long, 16 are reserved for the epoch
+   *           and 8 for recursive directory creation.
+   * @return base object id allocated against the transaction
+   */
+  public static long getObjectIdFromTxId(long epoch, long id) {
+    Preconditions.checkArgument(id <= MAX_TRXN_ID, "TransactionID " +
+        "exceeds max limit of " + MAX_TRXN_ID);
```

Comment:
I am thinking for this extreme case, user cannot write object anymore when TransactionID exceeds MAX_TRXN_ID, right? So what can we do for this, have to setup a new Ozone cluster to use?